.travis.yml

```diff
@@ -17,3 +17,4 @@ install: true
 script:
   - go run main.go download all
   - go test -coverprofile=cover.out -v ./...
+  - go install mvdan.cc/gofumpt && git ls-files '*.go' | grep -v testdata | xargs gofumpt -d -s
```

Comment:
I'm pretty sure `-d` will not error if there are changes. did you test this?

see what we do in openbank-services/.drone.yml

Reply:
You are correct, it does not error. I tested on Travis now :) - https://travis-ci.org/gunk/gunk/builds/653393558?utm_medium=notification&utm_source=github_status 

OK, I will do the `test -z` way

Reply:
with the `test-z` it does "error properly" - https://travis-ci.org/gunk/gunk/jobs/653396838?utm_medium=notification&utm_source=github_status . nice, thx